channel.js

```diff
@@ -810,6 +820,7 @@ function fastRequestDefaults(reqOpts) {
 function RequestDefaults(reqDefaults) {
     this.timeout = reqDefaults.timeout || 0;
     this.retryLimit = reqDefaults.retryLimit || 0;
+    this.maxRetryRatio = reqDefaults.maxRetryRatio || 1.0;
```

Comment:
Do we really want this parameter per request ? I suspect that this would be something configured for the entire serviceName.